### PR TITLE
[Toolchain] Update several information messages

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit changes the message contents.

ONE-vscode-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

**Review by llFreetimell — DISMISSED** [review]
LGTM

File: src/Job/Job.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ export type JobCallback = {
 export interface Job {
   jobType: Job.Type;
   name: string;
+  title?: string;
```
[the remark this answers is not in the document]
Comment (jyoungyun) [reply src/Job/Job.ts]: How about use `notiTitle`? The name `notificationTitle` is too long. :)

File: src/Toolchain/ToolchainProvider.ts [hunk 2]
```diff
@@ -221,15 +228,15 @@ export class ToolchainProvider implements vscode.TreeDataProvider<BaseNode> {
       return;
     }
 
-    Logger.info(this.tag, `Compile toolchain: ${activeToolchain.info.name}-${activeToolchain.info.version?.str()}`);
+    Logger.info(this.tag, `Run config file with ${activeToolchain.info.name}-${activeToolchain.info.version?.str()} toolchain.`);
     activeToolchainEnv.run(cfg, activeToolchain).then(() => {
-      vscode.window.showInformationMessage('Compile successfully');
-    }, () => this.error('Failed to compile model'));
+      vscode.window.showInformationMessage('Run onecc was successful.');
```
Comment (dayo09) [inline src/Toolchain/ToolchainProvider.ts:233]:
```suggestion
      vscode.window.showInformationMessage('Onecc has run successfully.');
```

File: src/Toolchain/ToolchainProvider.ts [hunk 3]
```diff
@@ -221,15 +228,15 @@ export class ToolchainProvider implements vscode.TreeDataProvider<BaseNode> {
...
+    }, () => this.error('Run onecc has failed.'));
```
Comment (dayo09) [inline src/Toolchain/ToolchainProvider.ts:234]:
```suggestion
    }, () => this.error('Running onecc has failed.'));
```

File: src/Toolchain/ToolchainProvider.ts [hunk 4]
```diff
@@ -221,15 +228,15 @@ export class ToolchainProvider implements vscode.TreeDataProvider<BaseNode> {
       return;
     }
 
-    Logger.info(this.tag, `Compile toolchain: ${activeToolchain.info.name}-${activeToolchain.info.version?.str()}`);
+    Logger.info(this.tag, `Run config file with ${activeToolchain.info.name}-${activeToolchain.info.version?.str()} toolchain.`);
```
Comment (dayo09) [inline src/Toolchain/ToolchainProvider.ts:231]: I think 'running onecc' sounds to make sense than 'running config file', so I suggest:

```suggestion
    Logger.info(this.tag, `Run onecc with ${cfg} cfg and ${activeToolchain.info.name}-${activeToolchain.info.version?.str()} toolchain.`);
```

File: src/Toolchain/ToolchainProvider.ts [hunk 5]
```diff
@@ -188,21 +188,28 @@ export class ToolchainProvider implements vscode.TreeDataProvider<BaseNode> {
...
-        Logger.info(this.tag, 'Default toolchain is cancelled.');
-        DefaultToolchain.getInstance().unset();
-      }
-      this.refresh();
-    });
+    gToolchainEnvMap[backendName]
+        .uninstall(tnode.toolchain)
+        .then(
+            () => {
+              vscode.window.showInformationMessage(`Uninstallation was successful.`);
```
Comment (dayo09) [inline src/Toolchain/ToolchainProvider.ts:203]: I think I heard below expression more, so I suggest: 

```suggestion
              vscode.window.showInformationMessage(`Successfully uninstalled.`);
```

File: src/Toolchain/ToolchainProvider.ts [hunk 6]
```diff
@@ -135,19 +135,19 @@ export class ToolchainProvider implements vscode.TreeDataProvider<BaseNode> {
   install() {
     const notifyInstalled = (toolchainEnv: ToolchainEnv, toolchain: Toolchain) => {
       const name = `${toolchain.info.name}-${toolchain.info.version ?.str()}`;
-      vscode.window.showInformationMessage(`Install ${name} successfully`);
+      vscode.window.showInformationMessage(`${name} installation was successful.`);
```
Comment (dayo09) [inline src/Toolchain/ToolchainProvider.ts:138]: How about this? To make sure that we are the subject.

```suggestion
      vscode.window.showInformationMessage(`Installed ${name} successfully.`);
```

File: src/Toolchain/DefaultToolchain.ts [hunk 7]
```diff
@@ -56,9 +56,10 @@ class DefaultToolchain {
     this._toolchainEnv = toolchainEnv;
     this._toolchain = toolchain;
     const name = `${toolchain.info.name}-${toolchain.info.version ?.str()}`;
-    vscode.window.showInformationMessage(`Set ${name} as a default toolchain.`, 'OK', `Instruction`)
+    vscode.window
+        .showInformationMessage(`${name} was set as a default toolchain.`, 'OK', `Instructions`)
```
Comment (dayo09) [inline src/Toolchain/DefaultToolchain.ts:60]: How about this?
I think most vscode buttons on information messages has "Do something" format. ex. Open something, View logs, Reload window, etc,,,

```suggestion
        .showInformationMessage(`${name} was set as a default toolchain.`, 'OK', `See Instructions`)
```

File: src/Toolchain/DefaultToolchain.ts [hunk 8]
```diff
@@ -56,9 +56,10 @@ class DefaultToolchain {
...
         .then((value) => {
-          if (value === 'Instruction') {
+          if (value === 'Instructions') {
```
Comment (dayo09) [inline src/Toolchain/DefaultToolchain.ts:62]: ditto
```suggestion
          if (value === 'See Instructions') {
```

**Review by hyunsik-yoon — APPROVED** [review]

**Review by dayo09 — APPROVED** [review]
Thanks!

**Review by llFreetimell — APPROVED** [review]

**Pull request merged**